contextutil/context.go

```diff
@@ -47,13 +49,19 @@ func Tenant(ctx context.Context) (config.TenantConfiguration, error) {
 }
 
 // CentContext updates a context with tenant info using the configstore, must only be used for api handlers
-func CentContext(ctx context.Context, config config.Service) (context.Context, error) {
-	// TODO [multi-tenancy] remove following and read the tenantID from the context
-	tc, err := config.GetAllTenants()
+func CentContext(ctx context.Context, cs config.Service) (context.Context, error) {
+	tcIDHex := ctx.Value(config.TenantKey).(string)
```

Comment:
Can we build a wrapper function (same as above) for loading the TenantKey?

Reply:
Sure we can but I'm curious what the use case would be? since this function is the only one needed by handlers to extract auth header.

Reply:
Just want to be consistent with every value in the context has a helper function to be retrieved abstracting the type check to the function itself and not the consumer.

Reply:
Just don't want that to be used by anything other than the handlers, having that kind of function gives the meaning that it's a common value that should be used, which is not the case as it comes from outside of the node, should be used with care, and any validation has to go here in this function in the future.

Reply:
I do not see a problem on that (for that particular case), if they have access to the TenantConfig they have access to the TenantID and the context is immutable.